BeVolt/Tests/Test_BSP_WDTimer.c

```diff
@@ -0,0 +1,52 @@
+#include "common.h"
+#include "config.h"
+#include "BSP_WDTimer.h"
+
+int main(){
+    uint32_t delay = 150000000;
```

Comment:
What unit is this in. Could we name it something like `delayMilliseconds` or `delayMicroseconds`?